Extensions/dist/extensions.js

```diff
@@ -157,3 +167,11 @@ function writeListFile(extensions) {
   });
   writePageFile('list.json', JSON.stringify(list));
 }
+
+if (require.main === module) {
```

Comment:
I mostly wanted to keep create-extensions-dist.sh working with no modification, although I can definitely get it to use gulp build:extensions and remove this.